dashboard-app/src/components/Charts/SimpleTable.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+
+
+const SimpleTable = (props) => {
```

Comment:
above each table/chart have a comment showing the structure of the data that will be coming in through props